Makefile

```diff
@@ -54,7 +54,7 @@ build-v4: .npminstall
 run:
 	@echo Starting redoc server
 
-	redoc-cli serve v4/html/static/mattermost-openapi-v4.yaml
+	@node_modules/redoc-cli/index.js serve v4/html/static/mattermost-openapi-v4.yaml
```

Comment:
Is it possible to leverage npx vs. hard-coding the path?

Reply:
ah neat! done.